### ISSUE TITLE
Link to Preface

### ISSUE THREAD

**Issue by gasyoun** [sanskrit-lexicon/VCP#3]
At http://www.sanskrit-lexicon.uni-koeln.de/ we now have B   L   A   M   D. That's much better than was before, it's quick and works like a charm. What if P would be added after for a link to https://github.com/sanskrit-lexicon/VCP/blob/master/Vachaspatyam-Preface_Tirupati2013.pdf preface. I do not have the scans on which these pages are based (that would be a needed addition), but the digital text for now would still serve a good service, do you agree? And what about abbreviations - for Vacaspatyam there is a list. Should we link to it from homepage? I guess no, but linking from inside would be needed. That's a quick fix, is it not?

**Comment by chandrasekharanr** [comment]
The preface https://github.com/sanskrit-lexicon/VCP/blob/master/Vachaspatyam-Preface_Tirupati2013.pdf is actually incomplete. Is there a way we can get the rest of the front matter for Vachaspatyam?

**Comment by gasyoun** [comment]
I guess you can scan the missing pages. Or find them on archive.org @chandrasekharanr ?

**Comment by chandrasekharanr** [comment]
Yes, that is an option. But the font in the original version is difficult to read, sometimes impossible. How do we get it in the unicode format as this one? :  https://github.com/sanskrit-lexicon/VCP/blob/master/Vachaspatyam-Preface_Tirupati2013.pdf

**Comment by gasyoun** [comment]
> font in the original version is difficult to read

Exactly.

> How do we get it in the unicode format as this one?

Do you know how to OCR and edit text?

**Comment by chandrasekharanr** [comment]
> Do you know how to OCR and edit text?
I do not know to OCR. But can help with editing the OCRed text if it is available. 

On a related note, why did not IITS-Koeln digitize the front matter for VCP in unicode? It only has a scanned pdf of the front matter here : https://www.sanskrit-lexicon.uni-koeln.de/scans/csldev/csldoc/build/dictionaries/vcp.html

**Comment by gasyoun** [comment]
> I do not know to OCR. 

https://ocr.sanskritdictionary.com

> But can help with editing the OCRed text if it is available.

You said that something is missing. I do not know what exactly is missing. How can I give it to you, if I do not know what. Try one scanned page and upload it the URL mentioned above, please.

> why did not IITS-Koeln digitize the front matter for VCP in unicode?

There is nobody left at IITS-Koeln. If you want to help, help is wanted. Out of 1.34 billion Indians let there be one who cares about digital dictionaries. Most of the text is English, not Devanagari, right? Chandra, download and install https://yadi.sk/d/a9uiwjz15DIp8A and https://yadi.sk/d/qfCa6yfPdGrEJ and give me your email (gmail preferred).

**Comment by chandrasekharanr** [comment]
> There is nobody left at IITS-Koeln. If you want to help, help is wanted. Out of 1.34 billion Indians let there be one who cares about digital dictionaries. Most of the text is English, not Devanagari, right? Chandra, download and install https://yadi.sk/d/a9uiwjz15DIp8A and https://yadi.sk/d/qfCa6yfPdGrEJ and give me your email (gmail preferred).

I can surely help. My gmail ID is the same as my github ID.

**Comment by drdhaval2785** [comment]
@chandrasekharanr 
Are you still interested in this work? I can arrange to get the OCR done for you.

**Comment by chandrasekharanr** [comment]
> 
> 
> @chandrasekharanr
> Are you still interested in this work? I can arrange to get the OCR done for you.

Yes, still interested.

**Comment by drdhaval2785** [comment]
Sorry, but I checked https://archive.org/details/vacaspatyam01tarkuoft and there is no bibliography of VCP. So there is nothing which I can do for you @chandrasekharanr .

**Comment by Andhrabharati** [comment]
@gasyoun 
I do not know if you got the text out of the Tirupati edition yourself and made the pdf, or it wass done by someone else for you.

But the point is that is more of the text in there, which got missed in the pdf file you had posted. (it was a blind copy/paste done by the person, without looking into the actual file content that resulted in this missing)

Of course, it has still some missings wrt to the print pages. I have filled it all myself.

@chandrasekharanr 
What is it that you want from the VCP?

**Comment by Andhrabharati** [comment]
> 
> 
> Sorry, but I checked https://archive.org/details/vacaspatyam01tarkuoft and there is no bibliography of VCP. So there is nothing which I can do for you @chandrasekharanr .

@drdhaval2785 

This person is not talking about the bibliography, but just asking for the front pages' (preface etc.) matter of VCP.

**Comment by chandrasekharanr** [comment]
> @chandrasekharanr
> What is it that you want from the VCP?
> 
> [**The VCP front pages contain much useful reference material (handy - ready reckoner); and even VS Apte has taken the material (but not fully, reason unknown!) from there to add in his Sanskrit-English Dictionary.**]

Yes, there are lot of useful reference material in the VCP front pages, nAmalingAnushAsanam for example. The unicode version of the entire front matter is useful.

**Comment by Andhrabharati** [comment]
@chandrasekharanr 

just to read/refer through (pdf), or to use it somewhere (text)?

**Comment by chandrasekharanr** [comment]
@Andhrabharati 
Mainly reference. Text will be useful if we can copy + paste in for example, email correspondence etc.
